pleskdistup/common/src/mounts.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os
+import typing
+
+
+def get_fstab_configuration_misorderings(configpath: str) -> typing.List[typing.Tuple[str, str]]:
+    """
+    Analyzes the fstab configuration file to find misorderings in mount points.
+    This function reads the fstab configuration file specified by `configpath` and checks for any misorderings
+    in the mount points. A misordering is defined as a mount point that appears before its parent directory
+    in the fstab file.
+    Args:
+        configpath (str): The path to the fstab configuration file.
+    Returns:
+        List[Tuple[str, str]]: A list of tuples where each tuple contains a misordered parent directory and
+        its corresponding mount point.
+    Example:
+        >>> get_fstab_configuration_misorderings('/etc/fstab')
+        [('/home', '/home/user'), ('/var', '/var/log')]
+    """
+
+    if not os.path.exists(configpath):
+        return []
+
+    mount_points_order: typing.Dict[str, int] = {}
```

Comment:
Just curious, is it still beneficial to give type hints for local variables?

Reply:
I think so, since because we automatically check types using [the mypy GitHub action](https://github.com/plesk/dist-upgrader/blob/main/.github/workflows/mypy.yml)